pkg/operators/cluster-machine-approver.go

```diff
@@ -0,0 +1,30 @@
+package operators
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/cluster-api-actuator-pkg/pkg/framework"
+)
+
+const (
+	cmaDeployment      = "machine-approver"
+	cmaClusterOperator = "machine-approver"
+)
+
+var _ = Describe("[Feature:Operators] Cluster Machine Approver deployment", func() {
+	client, err := framework.LoadClient()
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
I think this needs to be inside a BeforeEach

Reply:
Moved it within the it()